src/provider/constants.ts

```diff
@@ -15,7 +15,7 @@ export const Timeouts = {
     /**
      * Time service allows for a `window-created` event after a client expects the window to exist
      */
-    WINDOW_EXPECT_TO_SEEN: 100,
+    WINDOW_EXPECT_TO_SEEN: 250,
```

Comment:
interesting naming :)

Reply:
Yeah, reads a bit odd, but follows from the terminology in `Environment`. Was about to change to MAX_WINDOW_EXPECT_TO_SEEN_INTERVAL, but feel like that doesn't add anything that isn't implicit from it being under `Timeouts`